src/modal/test/modal.spec.js

```diff
@@ -833,6 +833,34 @@ describe('$uibModal', function () {
         expect($document.find('.modal-backdrop')).not.toHaveClass('fade');
       });
     });
+    
+    describe('container', function() {
+      it('should be added to body by default', function() {
+        var modal = open({template: '<div>Content</div>'});
+
+        expect($document).toHaveModalsOpen(1);
+        expect($document).toHaveModalOpenWithContent('Content', 'div');
+      });
+
+      it('should not be added to body if container is passed', function() {
+        var subElement = '<div clas="details-panel"></div>';
```

Comment:
Removed usage of class in testcases so that this PR won't give the impression that any element can be found.